js/solstice.modal.js

```diff
@@ -0,0 +1,22 @@
+/*!
+ * Copyright (c) 2020 Eclipse Foundation, Inc.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * Contributors:
+ *   Martin Lowe <martin.lowe@eclipse-foundation.org>
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ */
+(function($, document) {
+  $('#eclipsefdn-modal').on('show.bs.modal', function (event) {
+      var $init = $(event.relatedTarget);
+      var $modal = $(this);
+      $modal.find('h4.modal-title').text($init.attr('data-title'));
+      $modal.find('#modal-presenter').text($init.attr('data-presenter'));
```

Comment:
Why are we using id and sometimes it's a class?

Reply:
The ones that are classes are classic modal elements and have class structure to them. The ID ones are custom attributes that were added in. I can switch it to use classes instead if you like.